### DISCUSSION TITLE
Locating supply drops

### DISCUSSION THREAD

**Discussion by meyar** [TEHE-Studios/ExpandedHelicopterEvents#273] [Q&A] [answered]
Is there a way to locate supply drops after the UI bubblearrows disappear?

**Comment by Chuckleberry-Finn** [comment, marked as answer]
The bubble is intended to simulate seeing the parachute/crate dropping down. Once it's too low you'd need to find it by searching. There are flares dropped along with crates, so you should hear them.

**Reply by meyar** [reply]
Ah gotcha. I didn't notice the sound cue, but it makes sense. I think I saw somewhere else regarding crashes that in 42 you were gonna try adding crash markings on the map, will there be an option for supply drops by chance? I've missed two so far due to noticing them late.

**Reply by Chuckleberry-Finn** [reply]
Going to try and make temporary markers on the map, that you can make permanent/write yourself. Just haven't had time to sit down with this project as it will require alot of polish all over.
